### PR TITLE
Fix a test for overflow on table: captions aren't clipped, only the table box is.

### PR DESCRIPTION
**Pull request by bert-github**
Fix a test: 'Overflow' on a table element doesn't clip the caption, but only the table box. Changed the overflowing CAPTION to an overflowing TD.

Set an explicit line-height, for UAs with a default line height > 1.2.

Also added two missing LINK elements elsewhere.

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/250080171)

# Firefox (nightly) #
Testing web-platform-tests at revision 8b3dc839ef68c13e96296bd2fe3f08b9a8173aae
Using browser at version BuildID 20170704100254; SourceStamp fef489e8c2a193dde885adc48deb74cc883a5881
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>3 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/6463/css/CSS2/syntax/signed-numbers-001.xht">/css/CSS2/syntax/signed-numbers-001.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6463/css/CSS2/syntax/signed-numbers-002.xht">/css/CSS2/syntax/signed-numbers-002.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6463/css/CSS2/visufx/overflow-applies-to-001.xht">/css/CSS2/visufx/overflow-applies-to-001.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/250080172)

# Sauce (safari) #
Testing web-platform-tests at revision 8b3dc839ef68c13e96296bd2fe3f08b9a8173aae
Using browser at version 10.0
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>3 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/6463/css/CSS2/syntax/signed-numbers-001.xht">/css/CSS2/syntax/signed-numbers-001.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6463/css/CSS2/syntax/signed-numbers-002.xht">/css/CSS2/syntax/signed-numbers-002.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/6463/css/CSS2/visufx/overflow-applies-to-001.xht">/css/CSS2/visufx/overflow-applies-to-001.xht</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | PASS    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/250080180)

# Chrome (unstable) #
Testing web-platform-tests at revision 8b3dc839ef68c13e96296bd2fe3f08b9a8173aae

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/250080181)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision 8b3dc839ef68c13e96296bd2fe3f08b9a8173aae

File: css/CSS2/visufx/overflow-applies-to-001-ref.xht [hunk 1]
```diff
@@ -6,7 +6,8 @@
   <link rel="author" title="Bert Bos" href="mailto:bert@w3.org/" />
 
   <style type="text/css">
-  #container {width: 20em; background: green; text-align: center}
+  #container {width: 20em; background: green; text-align: center;
+    line-height: 1.2em}
```
Comment (dbaron) [inline css/CSS2/visufx/overflow-applies-to-001-ref.xht]: probably better to use 1.2 than 1.2em (here, and in the test)

File: css/CSS2/visufx/overflow-applies-to-001-ref.xht [hunk 2]
```diff
@@ -35,13 +36,17 @@
      </div>
 
      <div id="child3">
-       <img id="grandchild3" src="../support/1x1-green.png" alt="green" />
+       <img id="grandchild3" src="support/1x1-green.png" alt="green" />
```
Comment (dbaron) [inline css/CSS2/visufx/overflow-applies-to-001-ref.xht]: It seems like the old path works and the new path doesn't... or am I missing something?

**Review by dbaron — APPROVED** [review]
This looks good, with two comments (although I'm not sure about the image path...).

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18498)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/274977057)

File: css/CSS2/visufx/overflow-applies-to-001-ref.xht (hunk 2 above)
Comment (gsnedders) [reply css/CSS2/visufx/overflow-applies-to-001-ref.xht]: I expect the old path doesn't work in the built output and the new path doesn't work in the unbuilt input. Probably need *another* copy of those files in `support`. :/

**Comment by Ms2ger** [comment]
I landed the link additions in #7571. @bert-github, would you mind updating your branch?

**Pull request merged**